services/ui-src/src/components/export/ExportedEntityDetailsTable.tsx

```diff
@@ -135,7 +132,6 @@ export interface Props {
   showHintText?: boolean;
```

Comment:
can remove `showHintText` from `Props` since we only use that interface for `ExportedEntityDetailsTable`